styles.css

```diff
@@ -1,57 +1,57 @@
-body{
-    width: 375px;
-    align-items: center;
+body {
+  width: 375px;
+  align-items: center;
 }
 
-nav{
+nav {
   background-color: white;
-  margin: 10px
+  margin: 10px;
 }
 
-.header1{
-  display:flex;
+.header1 {
+  display: flex;
   justify-content: space-between;
 }
 
-.header2{
-  display:flex;
+.header2 {
+  display: flex;
   justify-content: space-between;
 }
 
-.menu-button{
+.menu-button {
   width: 40px;
   padding-right: 30px;
   height: 50px;
 }
 
-header{
+header {
   text-align: center;
-  background-image: url(../images/header-illsutration-mobile@2x.png)
+  background-image: url(../images/header-illsutration-mobile@2x.png);
 }
 
-h1{
-  color: #172B4D;
+h1 {
+  color: #172b4d;
   font-family: 'Inter', sans-serif;
   font-size: 45px;
   font-weight: bold;
```

Comment:
- [ ] I suggest you to use `font-weight:bold` to `font-weight: 700` in order to stick to the figma design. its good for your typography and designing skills.

Reply:
Done